src/NuGet.Clients/PackageManagement.UI/PackageManagementFormat.cs

```diff
@@ -101,7 +104,7 @@ public int SelectedPackageManagementFormat
         public void ApplyChanges()
         {
             _settings.SetValue(PackageManagementSection, DefaultPackageManagementFormatKey, SelectedPackageFormat.ToString(CultureInfo.InvariantCulture));
-            _settings.SetValue(PackageManagementSection, DoNotShowPackageManagementSelectionKey, DoNotShowDialogValue.ToString(CultureInfo.InvariantCulture));
+            _settings.SetValue(PackageManagementSection, DoNotShowPackageManagementSelectionKey, ShowDialogValue.Value.ToString(CultureInfo.InvariantCulture));
```

Comment:
There'll always be a Get before ApplyChanges which will always initialize it. So it will be guarded against null.